### PR TITLE
FISH-1654: Log To Console and Verbose mode.

### PR DESCRIPTION
**Pull request by rdebusscher**

### CODE REVIEW

File: docs/modules/ROOT/pages/documentation/payara-server/logging/log-to-console.adoc [hunk 1]
```diff
@@ -0,0 +1,34 @@
+[[log-to-console]]
+= Log To Console
+
```
Comment (AlanRoth) [inline docs/modules/ROOT/pages/documentation/payara-server/logging/log-to-console.adoc:3]: Add a introductory section that explains what Log To Console does.

File: docs/modules/ROOT/pages/documentation/payara-server/logging/log-to-console.adoc [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+The `Log to Console` option is only effective when the Server or the Instance is started with the verbose option.
+
+[source, shell]
+----
+asadmin start domain --verbose
+----
+
+When the server or domain is not started in verbose mode, the `java.util.ConsoleHandler` will not be active for logging although it is defined as handler in the _logging.properties_ file because there is no console attached with the process running the Payara server or Instance.
+
+When in verbose mode, the log to Console option will also be active, regardless of the initial value in the _logging.properties_ as otherwise, no log output would be visible on the console. Once started in verbose mode, you can use the Admin Console or the asadmin CLI to change the option.
```
Comment (AlanRoth) [inline docs/modules/ROOT/pages/documentation/payara-server/logging/log-to-console.adoc:13]: This sentence could be condensed a little and put under a 'Enabling/Disabling' subheading.

```suggestion
== Enabling
To enable the `Log to Console` option you simply have to start the Server or Instance with the verbose option.

[source, shell]
----
asadmin start domain --verbose
----

When using verbose mode, the log to Console option will also be active, regardless of the initial value in the _logging.properties_  file. Once started in verbose mode, you can use the Admin Console or the asadmin CLI to change this option.

NOTE: If Verbose mode is not used, the `java.util.ConsoleHandler` will not have a console attached to the process and will not be able to log to a console.

=== Disable Log To Console Using The Admin Console

To configure the `Log to Console` option using Admin Console, you can navigate to `Configurations > ${your-config} > Logger Settings > General > Log To File`

image:logging/log_to_file.png[Log to Console disabled]

=== Disable Log To File Using A `asadmin` Command

Use the following command to enable or disabled logging to the console:

[source, shell]
----
asadmin> set-log-attributes com.sun.enterprise.server.logging.GFFileHandler.logtoConsole=false
----
```

File: docs/modules/ROOT/pages/documentation/payara-server/logging/log-to-console.adoc [hunk 3]
```diff
@@ -0,0 +1,38 @@
+[[log-to-console]]
+= Log To Console
```
Comment (OndroMih) [inline docs/modules/ROOT/pages/documentation/payara-server/logging/log-to-console.adoc:2]:
```suggestion
= Logging To Console
```

File: docs/modules/ROOT/pages/documentation/payara-server/logging/log-to-console.adoc [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+The `Log to Console` option controls if the server writes the logging entries to the console. The property only has an effect when the server or instance is started in verbose mode.
+
+== Enabling
```
Comment (OndroMih) [inline docs/modules/ROOT/pages/documentation/payara-server/logging/log-to-console.adoc:6]:
```suggestion
[[enable]]
== Enable Logging To Console
```

File: docs/modules/ROOT/pages/documentation/payara-server/logging/log-to-console.adoc [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+----
+asadmin start domain --verbose
+----
+
+When the server or domain is not started in verbose mode, the `java.util.ConsoleHandler` will not be active for logging although it is defined as handler in the _logging.properties_ file because there is no console attached with the process running the Payara server or Instance.
+
+When in verbose mode, the log to Console option will also be active, regardless of the initial value in the _logging.properties_ as otherwise, no log output would be visible on the console. Once started in verbose mode, you can use the Admin Console or the asadmin CLI to change the option.
+
+[[using-web-admin-console]]
+== Using the Admin Console
```
Comment (OndroMih) [inline docs/modules/ROOT/pages/documentation/payara-server/logging/log-to-console.adoc:20]:
```suggestion
== Disable Logging To Console Using Admin Console
```

File: docs/modules/ROOT/pages/documentation/payara-server/logging/log-to-console.adoc [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+To configure `Log to Console` option using Admin Console:
+
+image:logging/log_to_file.png[Log to Console disabled]
+
+[[using-asadmin-utility]]
+== Using an `asadmin` command
```
Comment (OndroMih) [inline docs/modules/ROOT/pages/documentation/payara-server/logging/log-to-console.adoc:27]:
```suggestion
== Disable Logging To File Using Asadmin CLI
```
Current name is Asadmin CLI: https://payara.atlassian.net/wiki/spaces/MARK/pages/2387247181/Feature+Re-naming

**Review by OndroMih — CHANGES_REQUESTED** [review]
Some things missing from Alan's suggestions and I added also my suggestions.

File: docs/modules/ROOT/pages/documentation/payara-server/logging/log-to-console.adoc [hunk 7]
```diff
@@ -0,0 +1,38 @@
+[[log-to-console]]
+= Logging To Console
```
Comment (OndroMih) [inline docs/modules/ROOT/pages/documentation/payara-server/logging/log-to-console.adoc:2]:
```suggestion
= Log To Console
```

**Review by OndroMih — APPROVED** [review]

File: docs/modules/ROOT/pages/documentation/payara-server/logging/log-to-console.adoc [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+----
+asadmin start domain --verbose
+----
+
+When the server or domain is not started in verbose mode, the `java.util.ConsoleHandler` will not be active for logging although it is defined as handler in the _logging.properties_ file because there is no console attached with the process running the Payara server or Instance.
+
+When in verbose mode, the log to Console option will also be active, regardless of the initial value in the _logging.properties_ as otherwise, no log output would be visible on the console. Once started in verbose mode, you can use the Admin Console or the asadmin CLI to change the option.
+
+[[using-web-admin-console]]
+== Disable Logging To Console Using Admin Console
```
Comment (OndroMih) [inline docs/modules/ROOT/pages/documentation/payara-server/logging/log-to-console.adoc:21]:
```suggestion
== Configure Logging To Console Using Admin Console
```

File: docs/modules/ROOT/pages/documentation/payara-server/logging/log-to-console.adoc [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+
+To configure `Log to Console` option using Admin Console:
+
+image:logging/log_to_file.png[Log to Console disabled]
+
+[[using-asadmin-utility]]
+== Disable Logging To File Using Asadmin CLI
```
Comment (OndroMih) [inline docs/modules/ROOT/pages/documentation/payara-server/logging/log-to-console.adoc:28]:
```suggestion
== Configure Logging To File Using Asadmin CLI
```

**Review by OndroMih — APPROVED** [review]

File: docs/modules/ROOT/pages/documentation/payara-server/logging/log-to-console.adoc [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+
+[[using-web-admin-console]]
+== Configure Logging To Console Using Admin Console
+
+To configure `Log to Console` option using Admin Console:
+
+image:logging/log_to_file.png[Log to Console disabled]
+
+[[using-asadmin-utility]]
+== Configure Logging To File Using Asadmin CLI
```
Comment (OndroMih) [inline docs/modules/ROOT/pages/documentation/payara-server/logging/log-to-console.adoc:28]: I've fixed the typo (File -> Console) in this commit after merging this PR: https://github.com/payara/Payara-Community-Documentation/commit/b6520676f0995a9d55f3562badd5082188199693

**Pull request merged**